src/common/CommonUI/CommonUI.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
I think the naming for the DLL / project should be something more like Microsoft.PowerToys.Common.UI or something like that.  For exe naming and honestly even DLLs, i want to be a bit more crisp on what is a PowerToy DLL versus someone elses.